tests/Microsoft.Azure.Batch.SoftwareEntitlement.Tests/NodeEntitlementsBuilderTests.cs

```diff
@@ -271,5 +272,26 @@ public void WhenInvalidAddress_ReturnsErrorForAddress()
                 entitlement.Errors.Should().Contain(e => e.Contains("address"));
             }
         }
+
+        public class IssuerProperty : NodeEntitlementsBuilderTests
+        {
+            private readonly string _issuer = "https://account.region.batch.azure.test";
+
+            [Fact]
+            public void WhenMissing_BuildReturnsValue()
+            {
+                _commandLine.Issuer = string.Empty;
+                var result = NodeEntitlementsBuilder.Build(_commandLine);
+                result.HasValue.Should().BeTrue();
```

Comment:
I've reworked these a bit to assert that the correct default value is being set. Improved test naming as well.